core/src/main/scala/org/apache/spark/api/python/PythonRDD.scala

```diff
@@ -179,7 +179,9 @@ private[spark] object PythonRDD extends Logging {
     type UnrolledPartition = Array[ByteArray]
     val allPartitions: Array[UnrolledPartition] =
       sc.runJob(rdd, (x: Iterator[ByteArray]) => x.toArray, partitions.asScala.toSeq)
-    val flattenedPartition: UnrolledPartition = Array.concat(allPartitions: _*)
+
+    import org.apache.spark.util.ArrayImplicits._
```

Comment:
I think we can change them to top-level imports, then SPARK-45686 can reuse these imports.

Reply:
Thanks, done. Moved to top.